app/controllers/result_filters/location_controller.rb

```diff
@@ -8,13 +8,19 @@ def create
       if(!params[:l])
         flash[:error] = "Please choose an option"
         redirect_to location_path(params_without_unsupported_location_params)
+      elsif params[:l] == "3"
+        redirect_to provider_path(params_for_provider_search)
       else
         redirect_to results_path(params_without_unsupported_location_params)
       end
     end
 
   private
 
+    def params_for_provider_search
```

Comment:
FOR INFO: This is removed in https://trello.com/c/qeSWGWrJ/2783-l-location-filter-add-filter-page-by-locatiion. Should disappear once my or your branch is merged ...

Reply:
What's being done in its place?

Reply:
The params are now allowed, that's all